### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by bertinkambale**
- [ ] I Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- [ ] Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).\
- [ ] Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- [ ] I Followed our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- [ ] Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
- [ ] I should personalize the content of your page. Choose a topic that is different than the one in the original design.
- [ ] I should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
  - Layout: composition and space between elements.
- The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- You can use a CSS framework (for example Bootstrap) for styling, if you want to.
- You should build only these 2 pages:
  - The *home page*.
  - The *about page*.
- Each of these pages should have versions for 2 different screen sizes: 
  - Mobile: up to 768px wide.
  - Desktop: 768px or wider.
  

- [ ] - Interactions
  - Links
    - The *home page* should have a link in the menu to the *about page*.
    - The logo in the header links to the *home page*.
  - Mobile menu
    - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
    - There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
    - The mobile menu should have a close (X) button that closes the menu.
- Dynamic page
  - The section "Featured speakers" should be created dynamically in JavaScript.
  - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

- [ ] 

- [ ] lom link of my project:  https://www.loom.com/share/8139dc32be5147f1aea5a7d64e114727

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            <p class="headline1">Want to learn to drive each kind of cars? <br> We teach , Bus driver, chauffer,Delivery,Emergency medical technian,motorman and more</p>
+            <h2 class="headline2">Starting from 01.11.2022</h2>
+            <p class="address">@243 lagome, palace VIP, kinshasa (DRC)</p>
+        </section>
+        <section class="second-section" id="program">
+            <h2 class="service-heading" id="Programs">Programs</h2>
+            <hr class="line1">
+            <div class="cards">
+                <div class="card1">
+                    <i class="fa-solid fa-person-chalkboard fa-3x"></i>
```
Comment (tufoinnkuo10) [inline index.html:57]: - [ ] please kindly make your text in the cards well aligned like in the original version. this makes your work look more organized. great job.
yours
Screenshot (564)
Orginal design
Screenshot (563)_LI

File: style.css [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+  .card1:hover {
+    border: 1px solid #fff;
+  }
+
+  .fa-solid {
+    color: #fff;
+  }
+
+  .card1 h3 {
+    color: rgb(250, 46, 46);
```
Comment (tufoinnkuo10) [inline style.css:505]: - [ ] kindly use the original red color `\#EC5242`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @bertinkambale 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
- [ ] Please your loom video link is broken/Video refusing to play. kindly check and ensure it is the right one and the video plays as expected
- [ ] Please make use of GitHub Pages to deploy your site
- [ ] Also, kindly Ensure that A link to the deployed project is present in the README

Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 3]
```diff
@@ -0,0 +1,627 @@
...
+}
+
+.header2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 10vh;
+  background-color: #fff;
+  gap: 10%;
+}
```
Comment (zieeco) [inline style.css:34]: - According to the previous reviewer, this issue is not yet resolved. Kindly improve your design to align the content in this header int the home and about page to match the original design below:

*your design*
Screenshot from 2022-10-28 10-11-37

*original design*
Screenshot from 2022-10-28 10-09-14

File: README.md [hunk 4]
```diff
@@ -1 +1,60 @@
...
+- HTML
+- JavaScript
+
+## Live Demo
+
+[Live Demo Link](https://bertinkambale.github.io/Driving-school-capstone/)
+
+## Loom link
+
+[Loom link](https://www.loom.com/share/c3d1132f8c98432b897e32b2ba620170)
```
Comment (zieeco) [inline README.md:21]: # :point_right:  PRESENTATION VIDEO

- [ ] Great job adding the video as requested by the previous reviewer, however, as it is a requirement that the video length should not exceed 5 min, Please, kindly update the video to meet this part of the project requirement as currently, the [video length](https://www.loom.com/share/c3d1132f8c98432b897e32b2ba620170) is `5:45` which doesn't meet the project requirement.
- [ ] Although you mentioned the feature you're proud of, you're yet to show that piece of code in your code editor that you're proud of. please add this missing feature in the updated video. :pray:

File: index.html (hunk 1 above)
Comment (zieeco) [reply index.html:57]: - [ ]  This issue is well resolved :heavy_check_mark:

File: index.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+            <a href="#">logout</a>
+        </nav>
+        <nav class="header2">
+            <a class="logo" href="index.html"><img src="IMG/logo.jpg" height="50vh"></a>
+            <div class="hamburger">
+                <div class="hamburger1"></div>
+                <div class="hamburger2"></div>
+                <div class="hamburger3"></div>
+            </div>
+            <div class="menu">
```
[the remark this answers is not in the document]
Comment (zieeco) [reply index.html:33]: - [ ]  This issue is not yet resolved :x:

File: style.css (hunk 2 above)
Comment (zieeco) [reply style.css]: - [ ]  This issue is well resolved :heavy_check_mark:

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @bertinkambale ,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @bertinkambale , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 

#### Highlight
- Great project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**